src/containers/EnterpriseApp/index.jsx

```diff
@@ -71,9 +46,17 @@ class EnterpriseApp extends React.Component {
           />
           <Route exact path={`${baseUrl}/courses/:courseId`} component={CoursewarePage} />
           {features.DASHBOARD_V2 ? (
-            <Route exact path={`${baseUrl}/admin/learners/:slug?`} component={AdminPage} />
+            <Route
+              exact
+              path={`${baseUrl}/admin/learners/:actionSlug?`}
+              render={routeProps => <AdminPage {...routeProps} enterpriseSlug={enterpriseSlug} />}
```

Comment:
Would wrapping `AdminPage` with `withRouter` avoid needing to pass in `routeProps` and `enterpriseSlug`?